src/styles.scss

```diff
@@ -242,4 +237,104 @@ p code {
 
 .card-title{
   color: var(--textColor) !important;
+}
+
+.thmbCol {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+}
+
+.thmbCol2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.img-ht {
+  height: 190px;
+  width: 130px;
+  background-color: #ffffff;
+  border: 1px solid #efefef;
+}
+.wrap {
+  width: 130px !important;
+  height: 190px !important;
+  overflow: hidden;
+}
+
+.iframeCert {
```

Comment:
@chaitrali-r we have same class in common.css file with iframe1 name and same properties.  remove duplicate class